### ISSUE TITLE
xwayland fails to install due to missing lib/libfontenc.la, but is available in lib64

### ISSUE THREAD

**Issue by wardweistra** [skycocker/chromebrew#1606]
## Description
Installing xwayland fails on a missing `/usr/local/lib/libfontenc.la`. But `/usr/local/lib64/libfontenc.la` is available.

Installing xwayland:
```shell
chronos@localhost ~/Downloads/tools/R-3.4.3 $ crew install xwayland
xwayland: X server configured to (only) work with weston
https://x.org
version 1.19.3
No precompiled binary available for your platform, downloading source...
Archive downloaded
Unpacking archive, this may take awhile...
gcc (GCC) 4.9.4
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
```
[...]
```shell
libtool:   error: cannot find the library '/usr/local/lib/libfontenc.la' or unhandled argument '/usr/local/lib/libfontenc.la'
make[2]: *** [Makefile:694: Xwayland] Error 1
make[2]: Leaving directory '/usr/local/tmp/crew/xorg-server-1.19.3.tar.gz.dir/xorg-server-1.19.3/hw/xwayland'
make[1]: *** [Makefile:611: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/xorg-server-1.19.3.tar.gz.dir/xorg-server-1.19.3/hw'
make: *** [Makefile:778: all-recursive] Error 1
xwayland failed to install: `make -j4 V=0` exited with 2
```

Indeed it doesn't exist in /usr/local/**lib**:
```shell
chronos@localhost ~/Downloads/tools/R-3.4.3 $ ls /usr/local/lib | grep font
libfontconfig.la
libfontconfig.so
libfontconfig.so.1
libfontconfig.so.1.9.4
```

However, it does exist in /usr/local/**lib64**
```shell
chronos@localhost ~/Downloads/tools/R-3.4.3 $ ls /usr/local/lib64 | grep font
libfontenc.a
libfontenc.la
libfontenc.so
libfontenc.so.1
libfontenc.so.1.0.0
libXfont2.a
libXfont2.la
libXfont2.so
libXfont2.so.2
libXfont2.so.2.0.0
```

## Versions and system information
chromebrew: `crew --version`
`0.4.3`
ChromeOS: `cat /etc/lsb-release`
```shell
CHROMEOS_ARC_ANDROID_SDK_VERSION=25
CHROMEOS_ARC_VERSION=4446936
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_RELEASE_BOARD=eve-signed-mpkeys
CHROMEOS_RELEASE_BRANCH_NUMBER=77
CHROMEOS_RELEASE_BUILDER_PATH=eve-release/R62-9901.77.0
CHROMEOS_RELEASE_BUILD_NUMBER=9901
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=62
CHROMEOS_RELEASE_DESCRIPTION=9901.77.0 (Official Build) stable-channel eve 
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_VERSION=9901.77.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=9901.77.0
```
Model: YOUR MODEL HERE
Pixelbook
Architecture: `uname -m`
`x86_64`

## Addtional information
It seems like a reverse x86_64 situation of #1605. However setting the LD_LIBRARY_PATH to include both paths doesn't solve it.

**Comment by wardweistra** [comment]
Installing wayland btw, because when installing R without `--with-x=no` it isn't satisfied with the other X packages so far.

**Comment by wardweistra** [comment]
My adaptations of the hard way in #1605 also didn't have effect:
Adding `--libdir=#{CREW_LIB_PREFIX}` to `xwayland.rb` and installing with -s. `libfontenc` already had `--libdir=#{CREW_LIB_PREFIX}` there.

**Comment by uberhacker** [comment]
@wardweistra: Thank you for reporting this issue.  As you probably already know, the GUI is still very much a work in progress.  I imagine there are quite a few packages that need reconfigured to work correctly including xwayland.

**Comment by lovedheart** [comment]
Fixed with #2110
